src/corehost/cli/args.cpp

```diff
@@ -54,6 +54,17 @@ void setup_shared_store_paths(const hostpolicy_init_t& init, const pal::string_t
         append_path(&args->dotnet_shared_store, RUNTIME_STORE_DIRECTORY_NAME);
         append_path(&args->dotnet_shared_store, init.tfm.c_str());
     }
+    
+    // "dotnet.exe" relative shared store folder, adding the arch so that unzipped developer scenario works
+    if (init.host_mode == host_mode_t::muxer)
```

Comment:
Why does CLI attach an arch to the location? If the arch is not specified, then wont this merge into the earlier "shared store" location relative to dotnet.exe?